connector/spanmetricsconnector/connector.go

```diff
@@ -390,8 +390,32 @@ func (p *connectorImp) addExemplar(span ptrace.Span, duration float64, h metrics
 
 type resourceKey [16]byte
 
+func filterResourceAttr(attr pcommon.Map) pcommon.Map {
+	m := pcommon.NewMap()
+	attr.CopyTo(m)
+
+	s := []string{
+		conventions.AttributeServiceName,
+		conventions.AttributeTelemetrySDKName,
+		conventions.AttributeTelemetrySDKLanguage,
+	}
+
+	m.RemoveIf(func(k string, v pcommon.Value) bool {
+		for _, sv := range s {
```

Comment:
This is static so we can build the lookup at startup time as a member of `connectorImp`, perhaps when loading the config as suggested above. 

I also suggest we use a [sets](https://yourbasic.org/golang/implement-set/) with O(1) lookup time instead of a for loop.